Unix/installbuilder/conf/omiserver.conf

```diff
@@ -60,6 +60,6 @@ configfile=/etc/opt/omi/conf/omiserver.conf
 #NoSSLv3=false
 
 ## non-root feature parameters
-nonroot=true
-service=omiservice
-restartEngine=true
+#nonroot=true
+#service=omi
+#restartEngine=true
```

Comment:
Please create a bug for this.  I will investigate this issue.

As for the selector loop timeout, it's one second.